aksel.nav.no/website/sanity/schema/custom-components/InputWithCounter.tsx

```diff
@@ -13,9 +14,8 @@ export function InputWithCounter(
     <VStack gap="1">
       {props.renderDefault(props)}
       <Counter
-        // @ts-expect-error - maxLength is a custom prop not officially supported
         maxLength={schemaType?.options?.maxLength}
-        currentLength={value?.length ?? 0}
+        currentLength={typeof value === "string" ? value?.length ?? 0 : 0}
```

Comment:
Pretty sure all strings have a length? 🤔
```suggestion
        currentLength={typeof value === "string" ? value.length : 0}
```

Reply:
🧠